### PR TITLE
Feature/deployment

### PR DESCRIPTION
**Pull request by hb308**
# Description
Deployment will be documented in a read me file. However, the microservices have been deployed to an EKS cluster and a load balancer sits on this EKS cluster which handles the requests and routes it to the correct service based on the ingress controller. The only changes in files has been deploying to docker on push to main which is done through a github workflow. Also, the deployment points to ECR images in AWS

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +67,14 @@ To run the kubernetes cluster, run
...
 ```
 
+## Running from EKS
+
+To access the endpoints from a kubernetes cluster, you can access the endpoint at:
+
+```bash
+revenite.ada.and.sh
+```
+
```
Comment (ndeasy) [inline README.md:77]: Do we need to add any documentation about AWS and the keys required?

**Review by ndeasy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hb308) [reply README.md:77]: Yes I'm gonna get onto that tomorrow :)

Comment (hb308) [reply README.md:77]: It's gonna be quite a long piece of documentation. I'm going to include running it locally aswell as running it on the cloud etc.

Comment (ndeasy) [reply README.md:77]: Yeah makes sense to keep it separate to the readme

**Pull request merged**